plugins/Toolbox/src/Toolbox.py

```diff
@@ -517,7 +517,7 @@ def getNumberOfInstalledPackagesByAuthor(self, author_id: str) -> int:
     @pyqtSlot(str, result = int)
     def getTotalNumberOfPackagesByAuthor(self, author_id: str) -> int:
         count = 0
-        for package in self._metadata["materials_available"]:
+        for package in self._metadata["packages"]:
```

Comment:
This method is just used for the materials, so the "number of packages" means in fact "number of material packages". We should keep the code, but maybe change the name of the method or add documentation to avoid confusion.